census/templates/event.html

```diff
@@ -70,6 +58,19 @@
                 <span class="usa-hint line-height-body-1 font-body-2xs">{{form.is_census_equipped.help_text}}</span>
             </div>
           </div>
+
+          <label class="usa-label" for="{{form.start_datetime.id_for_label}}">Start time:</label>
+          <span class="usa-hint line-height-body-1 font-body-2xs">{{form.sstart_datetime.help_text}}</span>
+          <input type="time" name="start_datetime" required="" id="{{form.start_datetime.id_for_label}}">
+
+          <label class="usa-label" for="{{form.end_datetime.id_for_label}}">End time:</label>
+          <span class="usa-hint line-height-body-1 font-body-2xs">{{form.end_datetime.help_text}}</span>
+          <input type="time" name="end_datetime" required="" id="{{form.end_datetime.id_for_label}}">
+
+          <label class="usa-label" for="{{ form.recurrences.id_for_label }}">Recurrences:</label>
```

Comment:
I'd like to see the recurrence stuff off by default.  Can we have it displayed only with DEBUG or something?  We are getting close to sharing this with our partners and others (I have to be part a webinar for the Census Lab on Friday).  Since I cannot state what the rules are for the interaction of the start_datetime/end_datetime and recurrences, I would rather not have to try to explain it.

Reply:
FWIW, I'm pretty sure the recurrences field is working. I'll put it behind a feature flag. You can enable it by appending a query parameter `/submit/?enable_recurrence=1`